hack/update-deepcopy.sh

```diff
@@ -1,14 +1,15 @@
 #!/bin/bash
 
-set -o errexit
-set -o nounset
-set -o pipefail
+source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 
 SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/..
 CODEGEN_PKG=${CODEGEN_PKG:-$(cd ${SCRIPT_ROOT}; ls -d -1 ./vendor/k8s.io/code-generator 2>/dev/null || echo ../../../k8s.io/code-generator)}
 
+verify="${VERIFY:-}"
```

Comment:
in kube we have hack/update-codegen.sh and hack/verify-codegen.sh. Would be more self-explaining maybe.

Reply:
Yeah, but codegen in kube generates more than just deepcopies. When we'll do the same we can change. For now this make it explicit it's just the deepcopy.